docs/docs/contributing/contributing.md

```diff
@@ -0,0 +1,99 @@
+The mailing list is important for reach and perspective. You can challenge others
+that have different use cases for interaction with containers to give feedback on your idea.
+
+### 3. Support and Action
+
+Once discussion starts to die down, it's important to try and synthesize the content discussed into a concrete action. The maintainers will likely be doing this without thinking about it. For example: 
+
+ - *Documentation Action* if there is functionality you thought was missing and it actually existed? This calls for an improvement to the documentation. The maintainer can assign issues to developers to address your concern, and ask for your feedback.
+ - *Show Me Action*: The most likely result of you wanting a new thing will be a request for a dummy implementation. It doesn't need to be perfect, but you must *show* that your idea has feet, and solves the problem that you stated. 
+ - *Feature Request Action* It could be that your idea was succinct and strong, and a new feature is called for development. This likely will result in one or more developers (including you!) coming up with a plan of action. This is less likely an option, because changes to the specifications tend to be more conservative.
+
+Generally, you will want to find support for your action, whether it be advice or actual help from others in the list. We do this step before detailed feedback in the case that your idea doesn't need further investigation (already solved or being worked on, for example). What should never happen is anything that goes against the [code of conduct](https://github.com/opencontainers/org/blob/master/CODE_OF_CONDUCT.md). You should not be put down, ridiculed for your ideas, or otherwise have someone treat you poorly.
+
+### 4. Detailed Feedback
+
+After you've posted your idea to the list, I can almost guarantee you someone else will be interested or
+excited to give you feedback. The list (or a GitHub issue) itself isn't really the place to hash out the details.
+Your final post on the discussion might be to share a link to a more formal write up of your idea. You
+probably already have it written into a document, and it makes sense
+to share this writeup with others for feedback (Google Docs, GitHub markdown, HackMD, etc.). How long should you 
+get feedback? This is up to you. Make sure to give others credit and thanks for their contribution. If it's
+on GitHub, use an AUTHORS.md file. If it's a Google Doc, allow collaborators to write their name in a section,
+if desired. It's important to stress that the idea is a group effort. At this point, since you have others'
+attention, you might want to consider creating a working group to continue progress on your idea.
+What is a working group? It can be a small email list, a group of contributors subscribed to a common
```

Comment:
probably best to point them to the OCI resources here....

Reply:
Good idea, I'll add links to the Google Group, Slack, and org repo.